text/0000-Bundle-Load-API.md

```diff
@@ -50,10 +50,12 @@ The output of the API will be the function pointer type supplied by the API user
 /**
 * Retrieves the resolved symbol from bundle shared library and returns a function pointer associated with it
 * 
+* @param1 Handle to the Bundle's shared library
+* @param2 Name of the symbol
+*
 * @return A function pointer symbols of type T supplied by the user or nullptr if the library is not loaded
 * 
-* @throws std::runtime_error if the bundle is not started and active
-* @throws std::runtime_error if the symbol fetched is nullptr
+* @throws std::runtime_error if the bundle is invalid (i.e not started and active)
```

Comment:
An invalid bundle cannot be in the started or active state. I'd suggest using the previous text: `if the bundle is not started and active`